networktest.go

```diff
@@ -127,6 +131,7 @@ func newLightningNode(btcrpcConfig *rpcclient.ConnConfig, lndArgs []string) (*li
 			RPCUser: btcrpcConfig.User,
 			RPCPass: btcrpcConfig.Pass,
 		},
+		TrickleDelay: trickleDelay,
```

Comment:
Setting `TrickleDelay` here doesn't actually do anything as it isn't being passed into the arguments populated within `genArgs`. 

The default delay should be something on the order of 10s of seconds while within the integration tests, we'll use the `300ms` value in order to keep the tests snappy (especially when running on Travis).